src/modules/feed/pages/AddTransactionPage.js

```diff
@@ -126,13 +146,18 @@ export class AddTransactionPage extends Component {
 }
 
 AddTransactionPage.propTypes = {
-  isLoggedIn: PropTypes.bool.isRequired
+  isLoggedIn: PropTypes.bool.isRequired,
 };
 
 const mapStateToProps = state => ({
-  isLoggedIn: state.user.token !== null
+  isLoggedIn: state.user.token !== null,
+  transaction: state.feed.transactions.find(
```

Comment:
Have you considered using something like [reduxjs/reselect](https://github.com/reduxjs/reselect)?

Reply:
I've heard about this some time ago but never used it. I've created a selector for the isLoggedIn prop, it seems to me like you only create a selector on properties that will be reused across components? Or is it preferred to put every redux property binding in the selector file?